source/blog/2018-05-04-the-emberjs-times-issue-45.md

```diff
@@ -20,9 +20,11 @@ Here's a recap of what happened in Emberland these days ✨
 
 ---
 
-## [BROCCOLI JS TUTORIAL TITLE HERE](your-main-url-here)
+## [Listen to your Broccoli and it will tell you how to eat it](your-main-url-here)
```

Comment:
Cool title! I think it would be cool to add the url to the tutorial up here in the header as well, what do you think?

```
[Listen to your Broccoli and it will tell you how to eat it](http://www.oligriffiths.com/broccolijs/)
```